tests/test_pim.py

```diff
@@ -27,27 +28,66 @@ def test_pim_stage(fake_pim):
     logger.debug('test_pim_stage')
     pim = fake_pim
     # Should return to original position on unstage
-    pim.move('OUT', wait=True)
+    pim.remove()
     assert pim.removed
-    pim.stage()
-    pim.move('IN', wait=True)
+    pim.state.stage()
+    pim.insert()
     assert pim.inserted
-    pim.unstage()
+    pim.state.unstage()
     assert pim.removed
-    pim.move('IN', wait=True)
+    pim.insert()
     assert pim.inserted
-    pim.stage()
-    pim.move('OUT', wait=True)
+    pim.state.stage()
+    pim.remove()
     assert pim.removed
-    pim.unstage()
+    pim.state.unstage()
     assert pim.inserted
 
 
 @pytest.mark.timeout(5)
-def test_pim_det():
+def test_pim_init():
```

Comment:
This test is :fire:
If you're bored you can try to parametrize it

Reply:
Unfortunately, I don't have much time to be bored 😐